### PR TITLE
Global style fixes

### LINKED ISSUE #275: Global stying fixing
## Description
 
Currently since different people wrote different files the styling for each page is not quite consistent and does not entirely match the style guide, so really need a global styling fix.

## Acceptance Criteria

- [x] Styling across the whole app matches the style guide
- [x] Styling methods are consistent for similar components
- [x] Styling sheets use global variables for font weight, font size etc.
- [x] Remove unnecessary styling code
- [x] Move all stylings to CSS modules if possible
- [x] Dark mode styling is consistent with the style guide

## Implementation details

n/a

## Testing notes

n/a

### PR DESCRIPTION
**Pull request by rainasong**
Fixed almost all styling issues!!!

**Fixed styling issues for pages:**
- Homepage
- Create encounter page
- Create person page
- Settings page

**Fixed dark mode styling for components:**
- Navbar
- EncounterCard
- PersonCard
- EncounterSummaryCard
- PersonSummaryCard
- EncounterDrawer
- PersonDrawer
- EncounterDetails
- CustomButton

**Others:**
- Settings page dark mode toggle
- Homepage duplicated keys
- Few wordings
- Some UX flow design
- Colours.css variables

Fixes #275

### CODE REVIEW

**Review by lusicle — APPROVED** [review]
lgtm! 😎

File: forgettable-frontend/src/components/CustomButton/CustomButton.module.css [hunk 1]
```diff
@@ -21,7 +21,7 @@
 }
 
 .Text{
-    color: black;
+    color: var(--txt1);
     font-size: 18px;
```
Comment (ckar364) [inline forgettable-frontend/src/components/CustomButton/CustomButton.module.css:25]: Are we going to use global CSS for the font size here or keep it hard-coded?

File: forgettable-frontend/src/components/PersonCard/PersonCard.module.css [hunk 2]
```diff
@@ -62,7 +62,7 @@
     font-size: var(--text-card-body-large);
     /* line-height: 0; */
```
Comment (ckar364) [inline forgettable-frontend/src/components/PersonCard/PersonCard.module.css:63]: Should we remove the commented out "line-height"?

**Review by lin8231 — APPROVED** [review]
AMAZINGGG
SUPERSTAR

File: forgettable-frontend/src/components/CustomButton/CustomButton.module.css (hunk 1 above)
Comment (rainasong) [reply forgettable-frontend/src/components/CustomButton/CustomButton.module.css:25]: ahh sorry missed that! will do another fix!

File: forgettable-frontend/src/components/PersonCard/PersonCard.module.css (hunk 2 above)
Comment (rainasong) [reply forgettable-frontend/src/components/PersonCard/PersonCard.module.css:63]: Yep this will be fixed as well!

**Pull request merged**